app/code/Magento/Customer/Ui/Component/Listing/AttributeRepository.php

```diff
@@ -95,12 +95,13 @@ protected function getListForEntity(array $metadata, $entityTypeCode, MetadataMa
             if ($entityTypeCode == AddressMetadataInterface::ENTITY_TYPE_ADDRESS) {
                 $attributeCode = self::BILLING_ADDRESS_PREFIX . $attribute->getAttributeCode();
             }
+            $attributeOptions = $attribute->getOption();
             $attributes[$attributeCode] = [
                 AttributeMetadataInterface::ATTRIBUTE_CODE => $attributeCode,
                 AttributeMetadataInterface::FRONTEND_INPUT => $attribute->getFrontendInput(),
                 AttributeMetadataInterface::FRONTEND_LABEL => $attribute->getFrontendLabel(),
                 AttributeMetadataInterface::BACKEND_TYPE => $attribute->getBackendType(),
-                AttributeMetadataInterface::OPTIONS => $this->getOptionArray($attribute->getOptions()),
+                AttributeMetadataInterface::OPTIONS => is_array($attributeOptions) ? $this->getOptionArray($attributeOptions) : [],
```

Comment:
This line is too long. Magento code style standard requires lines to be no longer then 120 characters.